scipy/optimize/_differentialevolution.py

```diff
@@ -294,6 +295,53 @@ def differential_evolution(func, bounds, args=(), strategy='best1bin',
     iteration, rather than multiple times for all the population members; the
     interpreter overhead is reduced.
 
+    An outline of the strategies and their stengths and weaknesses is given by 
+    Qiang and Mitchell (2014) [8]_
```

Comment:
We should replace reference 3 on L252 (and elsewhere) with this reference, can you do that?